src/Sema.zig

```diff
@@ -598,6 +599,21 @@ pub const Block = struct {
         });
     }
 
+    fn addSwitchDispatch(
+        block: *Block,
+        target_block: Air.Inst.Index,
+        operand: Air.Inst.Ref,
+        comptime_known: bool,
+    ) error{OutOfMemory}!Air.Inst.Ref {
+        return block.addInst(.{
+            .tag = if (comptime_known) .switch_directbr else .switch_indirectbr,
```

Comment:
Do we actually need two instructions here? Backends can just handle the instruction differently depending on whether the operand is comptime-known.

Reply:
What function could I use on the backend to check if it's comptime known?

Reply:
Ok I suppose it's `Air.value`